pinc/stages.inc

```diff
@@ -284,10 +283,7 @@ function can_user_get_pages_in_project($username, $project, $round)
     if ($project->difficulty == 'beginner') {
         if ($round->is_a_mentee_round()) {
             if (!user_can_work_on_beginner_pages_in_round($round)) {
-                return
-                    _("You have reached your quota of pages from 'Beginners Only' projects in this round.")
-                    . " "
-                    . _("Perhaps you could try working on an EASY project.");
+                throw new BeginnersQuotaReachedException(_("You have reached your quota of pages from 'Beginners Only' projects in this round. Perhaps you could try working on an EASY project."));
```

Comment:
I don't have strong feelings about combining these sentences together vs keeping them apart, however this will impact the site translations for no discernible benefit. I defer to @srjfoo on how she would like to approach these changes.

Reply:
Oof, that's a good point.

I think I'm going to go for streamlining. I'm not sure why those strings were separated to begin with. Neither piece is used separately anywhere else. Can the code have line breaks to make for easier reading of the code?

Reply:
It seems you cannot escape newlines in strings in PHP like you can in C. But the DP_Code_Best_Practices wiki says you can break lines and the translators will ignore the extra spaces so I have done this.